interfaces/Portalicious/src/app/components/page-layout/page-layout.component.html

```diff
@@ -40,7 +40,13 @@
                 } @else {
                   <span class="font-normal">{{ parentTitle }}</span>
                 }
-                <i class="pi pi-chevron-right mx-2 text-sm"></i>
+                <i
+                  class="pi mx-2 text-sm"
+                  [ngClass]="{
+                    'pi-chevron-right': !rtlHelper.isRtl(),
+                    'pi-chevron-left': rtlHelper.isRtl(),
```

Comment:
Similar to the "polyfill"-approach mentioned above:

We could add a Tailwind-plugin/alias/class/thing like: `pi-chevron-start`
And have it output "the same as `pi-chevron-left`/`*-right`" depending on the `dir`-state.

That way, we don't have to "think about it"(and handle the RtlService, etc..) in our day-to-day-code...

Reply:
We could also create a "Direction-agnostic-chevron"-component...
So we can contain this logic... and don't have to repeat it...

I'll move this into: https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/34095